src/AzureAuth/Program.cs

```diff
@@ -97,6 +97,12 @@ private static void Main(string[] args)
             services.AddSingleton<IPublicClientAuth, PublicClientAuth>();
             services.AddSingleton<ILogger>(logger);
 
+            Console.CancelKeyPress += (_, _) =>
+            {
+                logger.LogWarning("Received SIGINT (Ctrl+C), exiting...");

```

Comment:
`logger?`?

Reply:
`logger` should not be null because it is initialized in line 93
```
            var logger = loggerFactory.CreateLogger("AzureAuth");
```

And the handler is registered after that.